akka-http-tests/src/test/java/akka/http/javadsl/server/JavaTestServer.java

```diff
@@ -142,11 +139,11 @@ public static void main(String[] args) throws InterruptedException {
 
   private void run() throws InterruptedException {
     final ActorSystem system = ActorSystem.create();
-    final ActorMaterializer mat = ActorMaterializer.create(system);
 
-    final Flow<HttpRequest, HttpResponse, ?> flow = createRoute().flow(system, mat);
     final CompletionStage<ServerBinding> binding =
-      Http.get(system).bindAndHandle(flow, ConnectHttp.toHost("127.0.0.1", 8080), mat);
+      Http.get(system)
```

Comment:
Look ma, no materializer necessary, no double usages of system, no `ConnectHttp`.